crates/turbopack-core/src/error.rs

```diff
@@ -0,0 +1,60 @@
+use std::fmt::{Display, Formatter, Result};
+
+pub struct PrettyPrintError<'a>(pub &'a anyhow::Error);
+
+impl<'a> Display for PrettyPrintError<'a> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        let mut i = 0;
+        let mut has_details = false;
+
+        let descriptions = self
+            .0
+            .chain()
+            .map(|cause| cause.to_string())
+            .collect::<Vec<_>>();
+
+        for description in &descriptions {
+            let hidden = description.starts_with("Execution of ");
```

Comment:
This is very fragile but I don't see a way around it with the way we use anyhow::Error everywhere. I would rather we try to downcast the actual errors to some known type/trait instead.

At least link to `task.rs` where this string is first created, and vice versa, so we don't forget to update both in sync.